runtime/src/main/java/org/corfudb/protocols/logprotocol/SMRRecord.java

```diff
@@ -43,83 +44,107 @@
     @Getter
     private ISerializer serializerType;
 
-    /** An undo record, which can be used to undo this method.
-     *
+    /**
+     * An undo record, which can be used to undo this method.
      */
     @Getter
     public transient Object undoRecord;
 
-    /** A flag indicating whether an undo record is present. Necessary
+    /**
+     * A flag indicating whether an undo record is present. Necessary
      * because undo records may be NULL.
      */
     @Getter
     public boolean undoable;
 
-    /** The upcall result, if present. */
+    /**
+     * The upcall result, if present.
+     */
     @Getter
     public transient Object upcallResult;
 
-    /** If there is an upcall result for this modification. */
+    /**
+     * If there is an upcall result for this modification.
+     */
     @Getter
     public transient boolean haveUpcallResult = false;
 
-    /** Set the upcall result for this entry. */
+    /**
+     * The global address this record is associated to.
+     */
+    @Getter
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651468)